Surf MVP and Coordinators Application.xctemplate/Xib/Makefile

```
@@ -0,0 +1,105 @@
+init:
+	# Install bundler if not installed
+	if ! gem spec bundler > /dev/null 2>&1; then\
+  		echo "bundler gem is not installed! Run gem install bundler to install it";\
+  		-sudo gem install bundler;\
```

Аналогично предыдущим
```suggestion
  		-sudo gem install bundle -v "1.17.3"r;\
```